src/hooks/useSubmitTxn.ts

```diff
@@ -18,7 +19,7 @@ import { useSelector } from 'react-redux'
 import activitySlice from '../store/activity/activitySlice'
 import { RootState } from '../store/rootReducer'
 import { useAppDispatch } from '../store/store'
-import { submitTransaction } from '../utils/appDataClient'
```

Comment:
We should be able to remove `submitTransaction` from `appDataClient` now, right?

Reply:
yes we can